executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -276,9 +276,6 @@ main_binary::B[3] = params[2];
 main_binary::operation_id[2] = main_binary::operation_id[3];
 main_binary::operation_id[1] = main_binary::operation_id[2];
 main_binary::operation_id[0] = main_binary::operation_id[1];
-main_binary::operation_id_next[-1] = main_binary::operation_id[0];
```

Comment:
I'm not sure how these changes appear. The order of processing should be the same as before. Maybe it's the assignments?